app/controllers/RdsController.scala

```diff
@@ -99,10 +99,13 @@ class RdsController @Inject()(cc: ControllerComponents)
       val statusJson = rdsAcknowledgeRequestValidationResult match {
         case JsSuccess(rdsRequest, _) =>
           try {
-            if (feedbackIDAndCorrelationIDMapping.contains(rdsRequest.feedbackId) &&
```

Comment:
Daz, please keep the code in if condition as it serves two purpose , reduces num of lines of code and it also check the id exist before looking up